src/Servers/Kestrel/Kestrel/test/KestrelConfigurationLoaderTests.cs

```diff
@@ -253,6 +253,38 @@ public void ConfigureEndpointDevelopmentCertificateGetsLoadedWhenPresent()
             }
         }
 
+        [Fact]
+        public void ConfigureEndpointDevelopmentCertificateIsCaseInsensitive()
```

Comment:
Since the change was in ConfigurationReader, you can probobly write a more targeted test in ConfigurationReaderTests. That would let you avoid some of this overhead like writing a cert to file.
https://github.com/dotnet/aspnetcore/blob/fa20111e47774b6c08f82a1f0c6373cc30e99a83/src/Servers/Kestrel/Kestrel/test/ConfigurationReaderTests.cs

Reply:
Good idea. I try to avoid file I/O (or really any I/O) in Kestrel tests if at all possible.